docs/documentation/release_notes/topics/22_0_0.adoc

```diff
@@ -86,6 +86,10 @@ The previous and now removed WildFly distribution provided a built-in vault prov
 
 In relation to the KeyStore Vault news, we also integrated Quarkus's recently released feature called KeyStore Config Source. This means that among the already existing configuration sources (CLI parameters, environment variables and files), you can now configure your Keycloak server via configuration properties stored in a Java keystore file. You can learn more about this feature in the https://www.keycloak.org/server/configuration[Configuration guide].
 
+= k8s.keycloak.org/v2alpha1 changes
+
+The are additional fields available in the keycloak.status to facilitate keycloak being a scalable resource.  There are also additional fields that make the status easier to interpret such as observedGeneration and condition observedGeneration and lastTransitionTime fields.  However the condition status field was also changed from a boolean to a string for conformance with standard Kubernetes conditions.  Please make sure any of your usage of this field is updated to expect the values "True", "False", or "Unknown", rather than true or false.
```

Comment:
```suggestion
The are additional fields available in the keycloak.status to facilitate keycloak being a scalable resource. There are also additional fields that make the status easier to interpret such as observedGeneration and condition observedGeneration and lastTransitionTime fields. However the condition status field was also changed from a boolean to a string for conformance with standard Kubernetes conditions. Please make sure any of your usage of this field is updated to expect the values "True", "False", or "Unknown", rather than true or false.
```